redaxo/src/core/lib/console/sync_assets.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * @package redaxo\core
+ * 
+ * @auth staabm
+ *
+ * @internal
+ */
+class rex_command_sync_assets extends rex_console_command
+{
+    protected function configure()
+    {
+        $this
+            ->setDescription('Sync files of /assets with addons/assets core/assets folders');

```

Comment:
- Via optionalem param auf ein/mehrere addons beschränken
-via —watch schalter dauerhaf laufen lassen

Reply:
Würde auch sagen, Ordner Erstellung via schalter erlauben.

Reply:
Ich glaube, ich würde Ordner immer synchronisieren und keinen Schalter dafür anbieten.

Reply:
> addons/assets core/assets

da fehlt ein Komma oder ein `and` oder irgend sowas.